dashboard/app/dsl/script_dsl.rb

```diff
@@ -45,6 +46,10 @@ def trophies(trophies_string)
     @trophies = ActiveRecord::ConnectionAdapters::Column::value_to_boolean(trophies_string)
   end
 
+  def wrapup_video(wrapup_video)
```

Comment:
this code is duplicated like 5 times now... if you want to get fancy you could do something like:

````
PROPERTY_NAMES = %w(description wrapup_video ...)
def method_missing(name, args)
  super unless PROPERTY_NAMES.include?(name)
  @properties[name] = args
end

def parse_output
  stage(nil)
  @properties
end
````